openshift/Makefile

```diff
@@ -45,3 +55,7 @@ test-experimental-e2e: ## Run the experimental e2e tests.
 	$(DIR)/operator-controller/build-test-registry.sh $(E2E_REGISTRY_NAMESPACE) $(E2E_REGISTRY_NAME) $(E2E_REGISTRY_IMAGE)
 	cd $(DIR)/../; \
 	go test $(DOWNSTREAM_EXPERIMENTAL_E2E_FLAGS) ./test/experimental-e2e/...;
+
+PHONY: go-build-local
+go-build-local:
+	$(MAKE) -f Makefile go-build-local
```

Comment:
Could you please let me know what would be the motivation to add this target?
It is not used here, only in the Dockerfile image